instrumentation/spring/starters/spring-boot-starter/build.gradle.kts

```diff
@@ -20,4 +20,13 @@ dependencies {
   api("io.opentelemetry:opentelemetry-exporter-otlp")
   api("io.opentelemetry:opentelemetry-sdk")
   api(project(":instrumentation-annotations"))
+
+  implementation("io.opentelemetry.contrib:opentelemetry-aws-resources:1.33.0-alpha") {
+    exclude("com.fasterxml.jackson.core", "jackson-core")
+    exclude("com.squareup.okhttp3", "okhttp")
+  }
+
+  implementation("io.opentelemetry.contrib:opentelemetry-gcp-resources:1.33.0-alpha") {
+    exclude("com.fasterxml.jackson.core", "jackson-core")
+  }
```

Comment:
why is it ok to exclude these dependencies?

Reply:
they are already contained in spring - and users expect the spring provided versions